### PR TITLE
chore: pin deps soak test

### PR DESCRIPTION
**Pull request by cmwylie19**
## Description

Pin dependencies to soak test action

## Related Issue

Fixes #

Relates to #

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Other (security config, docs update, etc)

## Checklist before merging

- [x] Test, docs, adr added or updated as needed
- [x] [Contributor Guide Steps](https://docs.pepr.dev/main/contribute/#submitting-a-pull-request) followed

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -73,7 +80,85 @@ jobs:
...
+          package-name: ${{ needs.slsa.outputs.package-name }}
+          package-download-name: ${{ needs.slsa.outputs.package-download-name }}
+          package-download-sha256: ${{ needs.slsa.outputs.package-download-sha256 }}
+          provenance-name: ${{ needs.slsa.outputs.provenance-name }}
+          provenance-download-name: ${{ needs.slsa.outputs.provenance-download-name }}
+          provenance-download-sha256: ${{ needs.slsa.outputs.provenance-download-sha256 }}
+
+
+jobs:
+  build:
```
Comment (btlghrants) [inline .github/workflows/release.yml]: Smells like there's some c&p chicanery going on here -- cuz doubled-up jobs blocks shouldn't be a thing, right?

**Review by btlghrants — CHANGES_REQUESTED** [review]
Looks like a bit too much got included..?

File: .github/workflows/release.yml (hunk 1 above)
Comment (cmwylie19) [reply .github/workflows/release.yml]: good catch, the paste was weird from the remediation tool

**Review by btlghrants — APPROVED** [review]
Looks great.

**Pull request merged**